examples/jenkins/pipeline/README.md

```diff
@@ -68,7 +69,7 @@ jenkins template represented by jenkinstemplate.json by running these commands a
 
     If you do not have a router, you can access jenkins directly via the service ip.  Determine the jenkins service ip ("oc get svc") and go to it in your browser on port 80.  Do not confuse it with the jenkins-jnlp service.
 
-    Login with the user name is `admin` and the password as recorded earlier.
+    Login with the user name is `admin` and any non-empty password.
```

Comment:
pending the discussion elsewhere, it could be either, but will adjust based on how that discussion pans out